interface/portraits/portraits_highelf.gfx

```diff
@@ -221,14 +221,14 @@ spriteTypes = {
 
```

Comment:
Also, I don't get what is this.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/81070838-de3e2300-8ef4-11ea-9a6c-94a27489dccb.png)

</details>

Reply:
looks like windows folder relics to me :D